### PR TITLE
Add contact component

### PR DESCRIPTION
**Pull request by WebDevRaj**
Adding new contact component with 2 variants - 
Screenshot (37)
Screenshot (38)

### CODE REVIEW

File: src/modules/resume/contact/variants/basic/properties.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  theme: {
+    name: 'Theme',
+    type: propertyTypes.THEME,
+    value: themes.light.iceWhite,
+    required: true
+  },
+  phoneNo: {
+    name: 'Phone No',
+    type: propertyTypes.NUMBER,
+    value: '9999999999',
```
Comment (RisingGeek) [inline src/modules/resume/contact/variants/basic/properties.js:14]: Value should be a number not string

File: src/modules/resume/contact/variants/basic/render.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+) => {
+  const styles = useStyles();
+
+  return (
+    <div style={{
+      ...styles.root,
+      backgroundColor: theme.value.color,
+      color: theme.value.contrast
+    }}>
+      <div><strong>{phoneNo.name}</strong> : {phoneNo.value}</div>
```
Comment (RisingGeek) [inline src/modules/resume/contact/variants/basic/render.js:27]: Rule violated
https://github.com/MSK-Documents/msk-ui/blob/38474324bfb46176084499d1b226ba6ddd6d484e/src/modules/resume/sample/variants/basic/render.js#L28

File: src/modules/resume/contact/variants/iconic/properties.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  theme: {
+    name: 'Theme',
+    type: propertyTypes.THEME,
+    value: themes.dark.shadowPurple,
+    required: true
+  },
+  phoneNo: {
+    name: 'Phone No',
+    type: propertyTypes.NUMBER,
+    value: '9999999999',
```
Comment (RisingGeek) [inline src/modules/resume/contact/variants/iconic/properties.js:14]: same here

File: src/modules/resume/contact/variants/iconic/render.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+) => {
+  const styles = useStyles();
+
+  return (
+    <div style={{
+      ...styles.root,
+      backgroundColor: theme.value.color,
+      color: theme.value.contrast
+    }}>
+      <p style={{...styles.item}}><PhoneIcon fontSize="small" /><span style={{...styles.itemValue}}>{phoneNo.value}</span></p>
```
Comment (RisingGeek) [inline src/modules/resume/contact/variants/iconic/render.js:37]: No need of spread operator

File: src/modules/resume/contact/variants/iconic/render.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PhoneIcon from '@material-ui/icons/Phone';
```
Comment (RisingGeek) [inline src/modules/resume/contact/variants/iconic/render.js:2]: Rule violated
https://github.com/MSK-Documents/msk-ui/blob/38474324bfb46176084499d1b226ba6ddd6d484e/src/modules/resume/sample/variants/basic/render.js#L20
Use svg or url of icon instead

File: src/modules/resume/contact/variants/basic/properties.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    required: true
+  },
+  emailId: {
+    name: 'Email ID',
+    type: propertyTypes.EMAIL,
+    value: 'Email ID',
```
Comment (rohanJa) [inline src/modules/resume/contact/variants/basic/properties.js:20]: Change value as type is an email id so value (which act as a placeholder) should be a example id like ` user@gmail.com`.

File: src/modules/resume/contact/variants/basic/properties.js (hunk 1 above)
Comment (WebDevRaj) [reply src/modules/resume/contact/variants/basic/properties.js:14]: What if there are special characters in the number like '+91' ?

Comment (WebDevRaj) [reply src/modules/resume/contact/variants/basic/properties.js:14]: Please check the properties of the 'cover' component, I used that as a reference.

Comment (ManojSatishkumar) [reply src/modules/resume/contact/variants/basic/properties.js:14]: Let's use number itself.
We will fix cover component.

Comment (WebDevRaj) [reply src/modules/resume/contact/variants/basic/properties.js:14]: OK, but shouldn't it be STRING to allow for prefix like '+91' ?

Comment (RisingGeek) [reply src/modules/resume/contact/variants/basic/properties.js]: Correct, should there be a property type named MOBILE?

File: src/modules/resume/contact/variants/basic/render.js (hunk 2 above)
Comment (WebDevRaj) [reply src/modules/resume/contact/variants/basic/render.js]: fixed

File: src/modules/resume/contact/variants/iconic/render.js (hunk 5 above)
Comment (WebDevRaj) [reply src/modules/resume/contact/variants/iconic/render.js]: added svg

File: src/modules/resume/contact/variants/iconic/render.js (hunk 4 above)
Comment (WebDevRaj) [reply src/modules/resume/contact/variants/iconic/render.js]: fixed

File: src/modules/resume/contact/variants/basic/properties.js (hunk 1 above)
Comment (WebDevRaj) [reply src/modules/resume/contact/variants/basic/properties.js]: I have kept it as number for now. We can fix it later maybe.

Comment (RisingGeek) [reply src/modules/resume/contact/variants/basic/properties.js]: Okay

File: src/modules/resume/contact/metadata.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+const metadata = {
+  id: 'resume-contact',
+  name: "Contact",
+  description: "Your contact information."
+}
+
+export default metadata;
```
Comment (AKAspanion) [inline src/modules/resume/contact/metadata.js:8]: No newline at the end.
Format all the files using prettier.

File: src/modules/resume/contact/variants/basic/properties.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  city: {
+    name: 'Current City',
+    type: propertyTypes.STRING,
+    value: 'City',
+    required: true
+  },
+  website: {
+    name: 'Website',
+    type: propertyTypes.STRING,
```
Comment (AKAspanion) [inline src/modules/resume/contact/variants/basic/properties.js:31]: This should be a URL type.

File: src/modules/resume/contact/variants/basic/render.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+const useStyles = (theme) => ({
```
Comment (AKAspanion) [inline src/modules/resume/contact/variants/basic/render.js:3]: Useless 'theme' argument.

**Comment by AKAspanion** [comment]
Please resolve conflicts.

**Comment by WebDevRaj** [comment]
> Please resolve conflicts.

Done

**Pull request merged**